.circleci/config.yml

```diff
@@ -1,12 +1,67 @@
 version: 2.1
 
-jobs:
-  build:
+commands:
+  get_test_results_path:
+    parameters:
+      path:
+        type: string
+        default: /tmp/test-results
+    steps:
+      - run: echo << parameters.path >>
```

Comment:
I think it's not possible to use this in the way it's intended. When we `run` a command, it happens in isolation in its own shell, so we can't dynamically read the output from executed commands in this way. Commands themselves are are used in the same way the `run` step is used, and can themselves contain multiple `run` steps, or references to other commands in their place.